### PR TITLE
Use enum properly to directly access kConfig struct

### PR DESCRIPTION
**Pull request by psiegl**

### CODE REVIEW

File: src/cpuinfo_aarch64.c [hunk 1]
```diff
@@ -113,29 +116,15 @@ int GetAarch64FeaturesEnumValue(const Aarch64Features* features,
     case AARCH64_CRC32:
       return features->crc32;
     case AARCH64_LAST_:
+    default:
```
Comment (gchatelet) [inline src/cpuinfo_aarch64.c]: Remove the default case here and in all the switches to allow the compiler to warn about missing enum case.

Comment (psiegl) [reply src/cpuinfo_aarch64.c]: fixed

File: src/cpuinfo_aarch64.c [hunk 2]
```diff
@@ -119,23 +122,8 @@ int GetAarch64FeaturesEnumValue(const Aarch64Features* features,
...
-      return "sha2";
-    case AARCH64_CRC32:
-      return "crc32";
-    case AARCH64_LAST_:
-      break;
-  }
-  return "unknown feature";
+  if(value >= kConfigsSize)
+    return "unknown feature";
+  else
```
Comment (gchatelet) [inline src/cpuinfo_aarch64.c]: Ha also can you remove the `else` here and below?
Sorry I should have mentioned it in the previous review...

Comment (psiegl) [reply src/cpuinfo_aarch64.c]: fixed

**Comment by gchatelet** [comment]
Thx!

**Pull request merged**